### ISSUE TITLE
exceptions being thrown when hovering over clinical data KM plot in BRCA 2013_04_21 for 3.0

### ISSUE THREAD

**Issue by ngehlenborg** [Caleydo/caleydo#1290]
To reproduce:
1. BRCA 2013_04_21
2. in first step directly add clinical variable and select patient.daystobirth from list
3. hover over KM plot until you get highlighting

I noticed that the selection info doesn't show anything, even when I select the brick.

I can't tell if this has any effect on the stability of the application but we should probably handle this.

```
!ENTRY org.eclipse.ui 4 0 2013-06-18 11:22:42.654
!MESSAGE Unhandled event loop exception
!STACK 0
org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String)
    at org.eclipse.swt.SWT.error(SWT.java:4361)
    at org.eclipse.swt.SWT.error(SWT.java:4276)
    at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:138)
    at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3946)
    at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3623)
    at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$9.run(PartRenderingEngine.java:1053)
    at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
    at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:942)
    at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:86)
    at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:588)
    at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
    at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:543)
    at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
    at org.caleydo.core.internal.Application.start(Application.java:87)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:353)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:180)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:601)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:629)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:584)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1438)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1414)
Caused by: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
    at org.caleydo.view.info.selection.RcpSelectionInfoView.createItems(RcpSelectionInfoView.java:238)
    at org.caleydo.view.info.selection.RcpSelectionInfoView.access$4(RcpSelectionInfoView.java:228)
    at org.caleydo.view.info.selection.RcpSelectionInfoView$3.run(RcpSelectionInfoView.java:213)
    at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
    at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
    ... 24 more
```

I also noticed this:

```
!ENTRY IDMappingManager for TCGA_SAMPLE_GROUP 1 0 2013-06-18 11:22:44.428
!MESSAGE No mapping found between [IDType of TCGA_SAMPLE_GROUP: TCGA_SAMPLE_GROUP_primary] and [IDType of TCGA_SAMPLE_GROUP: group_record_org.caleydo.datadomain.generic_9_311069349]
```

**Comment by alexsb** [comment]
Is this resolved with #1287 - I can't reproduce that anymore?

**Comment by ngehlenborg** [comment]
Yes, closing.
